test/e2e/exists_test.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func TestExistsOpWithPlaceholderAnn(t *testing.T) {
+	env := BuildEnv(t)
+	logger := Logger{}
+	kapp := Kapp{t, env.Namespace, env.KappBinaryPath, logger}
+	clientset, err := getKubernetesClientset()
```

Comment:
Should we deploy the external resource using `kapp` with a different app name? I think that will make it simpler. Thoughts?

Reply:
That does make sense